bfh/fields.py

```diff
@@ -90,6 +98,13 @@ def field_name(self):
     def field_name(self, value):
         self._field_name = value
 
+    @property
+    def default(self):
+        return self._default() if callable(self._default) else self._default
+
+    @default.setter
+    def default(self, value):
+        self._default = value
```

Comment:
Pep8. 2 blank lines. Not sure what editor you use but you can probably find a plugin that will lint for you automagically.